rnd/autogpt_builder/src/app/layout.tsx

```diff
@@ -27,10 +28,18 @@ function ThemeProvider({ children, ...props }: ThemeProviderProps) {
 const NavBar = () => (
   <nav className="bg-white dark:bg-slate-800 p-4 flex justify-between items-center shadow">
     <div className="flex space-x-4">
-      <Link href="/monitor" className={buttonVariants({ variant: "ghost" })}>Monitor</Link>
-      <Link href="/build" className={buttonVariants({ variant: "ghost" })}>Build</Link>
-      <Link href="/backtrack" className={buttonVariants({ variant: "ghost" })}>Backtrack</Link>
-      <Link href="/explore" className={buttonVariants({ variant: "ghost" })}>Explore</Link>
+      <Link href="/monitor" className={buttonVariants({ variant: "ghost" })}>
+        <TimerIcon className="mr-1" /> Monitor
+      </Link>
+      <Link href="/build" className={buttonVariants({ variant: "ghost" })}>
+        <Pencil1Icon className="mr-1" /> Build
+      </Link>
+      {/* <Link href="/backtrack" className={buttonVariants({ variant: "ghost" })}>
```

Comment:
can we remove the commented out stuff altogether?

Reply:
sure